src/index.css

```diff
@@ -1,27 +1,306 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100&display=swap');
+
 body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  background-color: whitesmoke;
+}
+
+header {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  background-color: white;
+  display: -webkit-box;
+  display: flex;
+  font-family: 'Montserrat Bold', sans-serif;
+  height: 5.938rem;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin: 0 auto;
+}
+
+header .header-text {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  display: -webkit-box;
+  margin-left: 3rem;
+}
+
+header .header-text > * {
+  margin-right: 1.5rem;
+}
+
+header .bookstore-cms {
+  margin-left: 4rem;
+  color: #0290ff;
+  font-size: 1.875rem;
+  font-weight: bold;
+}
+
+header .books {
+  color: #121212;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+}
+
+header .categories {
+  color: #121212;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  opacity: 0.5;
+}
+
+header .icon {
+  margin-right: 8rem;
+  color: #0290ff;
+  background-color: white;
+  border-radius: 50%;
+  border: 1px solid #e5e7eb;
+  padding: 10px;
+}
+
+.select-dropdown {
+  margin: 1rem auto;
+  width: 100%;
+}
+
+.select-dropdown-2,
+.form-submit {
+  width: 18%;
+}
+
+.book-input {
+  width: 60%;
+  background-color: #fff;
+  border-color: #e5e7eb;
+  border-radius: 4px;
+  border-style: solid;
+  border-width: 1px;
+  line-height: 24px;
+  padding: 12px 16px;
+}
+
+.lesson-panel {
+  width: 70rem;
+  height: 12rem;
+  margin: 1rem  0 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  border: solid 1px rgb(240, 240, 240);
+}
+
+.book-info {
+  grid-area: auto / span 3 / auto / span 3;
+  line-height: 24px;
+  text-align: left;
+}
+
+.category {
+  color: #121212;
+  font-family: 'Montserrat Bold', sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: left;
+  text-transform: capitalize;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_11_4_2021_2-08-53 AM](https://user-images.githubusercontent.com/45217364/140236762-006972cf-0f80-4e08-a646-011d763e8f4d.png) |
   -  The app should not be full-screen. It must be like a card in the center of the screen just like it is on the mockup design. Kindly fix it.
   - The selector should be in the toolbar, that's what that category text in the toolbar is there for. it is there to represent the category selector.
   - Toolbar's menu and icons, books list items and the create book form must be horizontally aligned, just like there are in mockup design.